File: Modul-1/C/PRAK109-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
    int jumlahPasukanYuZhong;
    printf("Jumlah Pasukan yang dibawa YuZhong: ");
    scanf("%d", &jumlahPasukanYuZhong);
    int jumlahPahlawan;
    printf("Jumlah Pahlawan: ");
    scanf("%d", &jumlahPahlawan);
    int jumlahPasukanYangHarusDikalahkanSetiapPahlawan;

    jumlahPasukanYangHarusDikalahkanSetiapPahlawan = jumlahPasukanYuZhong / jumlahPahlawan;

    printf("Jumlah pasukan yang harus dikalahkan setiap pahlawan adalah %d pasukan\n", jumlahPasukanYangHarusDikalahkanSetiapPahlawan);

    return 0;
}

File: Modul-1/python/PRAK109-2310817110006-Adrian_Bintang_Saputera.py
jumlah_pasukan_yu_zhong = int(input("Jumlah Pasukan yang dibawa YuZhong: "))
jumlah_pahlawan = int(input("Jumlah Pahlawan: "))

jumlah_pasukan_yang_harus_dikalahkan_setiap_pahlawan = (
    jumlah_pasukan_yu_zhong // jumlah_pahlawan
)

print(
    f"Jumlah pasukan yang harus dikalahkan setiap pahlawan adalah {jumlah_pasukan_yang_harus_dikalahkan_setiap_pahlawan} pasukan"
)
